<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>QR Text:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theQrText">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>geo:27.9282,-80.547</string>
    </property>
   </widget>
   <widget class="QPushButton" name="theGenerateButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QListWidget" name="theSerialRaw">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>1351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>370</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Radio Port:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="theRadioPortList">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="theOpenRadioButton">
       <property name="text">
        <string>Open Radio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="theBatteryBar">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="theBatteryVoltsLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>--- Volts</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS Fix:</string>
    </property>
   </widget>
   <widget class="QLabel" name="theGpsFixLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DOP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theDopValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Sats:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="theNumSatsValue">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1506</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
